_posts/2022-12-05-Introducing-Identity.markdown

```diff
@@ -0,0 +1,40 @@
+# Introducing Identity for OpenSearch
+
+OpenSearch handles critical data; how that data is permissioned and how a cluster is operated is a Big Deal™.  Consider the question "Who can delete all the indices in the cluster?", in the core of OpenSearch, the answer is whoever can make a request to the cluster.  
+
+Within the OpenSearch Project, the Security plugin allows finer levels of access control providing a more satisfying answer.  Security uses the plugin architecture of OpenSearch that forces concepts around identity to be optional in core, plugins, and extensions components.  Optional security is not enough for critical data, we will do to do better.
+
+Let’s permeate identity in every aspect of OpenSearch – enabling security by default - everywhere.
+
+## Identity is more than a user
+In this blog post we use the term  *Subject* that includes users and non-person entities.  Following NIST's defintion Subjects are '[a]n individual, process, or device that causes information to flow among objects or change to the system state.'
+
+Among the OpenSearch-Project's scenario there is monitoring, data management, replication, machine learning, and notificatios.  The identities of thse systems and process need a common description as a Subject.
+
+## What are OpenSearch’s identities?
+In OpenSearch systems, interaction with a cluster should be clearly associated with an identity. Sources of interaction include:
+- Users make REST API calls in interactive sessions
+- Node-to-node communication in a cluster
+- Background jobs interactions
+- Plugins in-process interactions
+- Extensions actions extra-process interactions
+
+![image](https://user-images.githubusercontent.com/2754967/201424268-5687f162-4857-424b-96b8-ca8aff5f649a.png)
+
+## Why introduce identity into the core of OpenSearch?
+By knowing who is performing actions we can check if that user is Authorized [3].  Plugins and extensions can only do actions that have been permitted to their identity.  The existing security plugin does not have the ability to control what other plugins can do.

```

Comment:
Where is the footnote for this: "Authorized [3]"? A separate file from this one?

Reply:
Heavily rewritten!